dockerfiles/base/scripts/base/commands/cmd_config.sh

```diff
@@ -116,6 +116,7 @@ generate_configuration_with_puppet() {
     # Handle override/addon
     if [ -d "/repo/dockerfiles/init/addon" ]; then
       WRITE_PARAMETERS+=" -v \"${CHE_HOST_DEVELOPMENT_REPO}/dockerfiles/init/addon/addon.pp\":/etc/puppet/manifests/addon.pp:ro"
+      WRITE_PARAMETERS+=" -v \"${CHE_HOST_DEVELOPMENT_REPO}/dockerfiles/init/addon/modules/\":/etc/puppet/addon/:ro"
```

Comment:
we should add a check if /repo/dockerfiles/init/addon/modules exists before adding the mount (so it's optional in repository that don't need modules)


also is it working if we add module directly in /etc/puppet/modules ? like /etc/puppet/modules/addon

so we don't need the change /etc/puppet/modules/:/etc/puppet/addon/